src/ape/tag/mod.rs

```diff
@@ -74,6 +74,11 @@ pub struct ApeTag {
 }
 
 impl ApeTag {
+	/// Returns the number of items in the tag
+	pub fn len(&self) -> usize {
+		self.items.len()
+	}
+
```

Comment:
```suggestion
```
I just added `TagExt::len` in #122, I thought this already existed. Thanks for the reminder.